less/index.less

```diff
@@ -57,3 +57,142 @@ html, body {
     height: 100%;
 }
 
+// Hi Patty, lets put our variables here.
+@font-stack: Verdana, "sans-serif";
+@first-color:blue;
+@second-color: black;
+@third-color: @blizzard-blue;
```

Comment:
these variables  could be a bit more descriptive.
think of what theyre bewing used for, for example they could be
top-color, bottom-color, normal-background